### DISCUSSION TITLE
Release Date for 2.5.0?

### DISCUSSION THREAD

**Discussion by andrewhosack** [sqlkata/querybuilder#625] [General]
Hey guys,

I'm working on an application that supports both MSSQL and Oracle databases. I love what SqlKata is doing and what it provides to my toolset! I found a bug in the oracle Insert Many functionality a couple of weeks ago, and I see that this issue has now been merged into Main Branch and I'll assume will be included in the next release! I was hoping to know if there is any indication on when the next release will be, that includes this fix? Not to rush you, but to better plan my own products development and release.

Thank you

**Comment by ahmad-moussawi** [comment]
Hi @andrewhosack, thanks for reaching out, we plan the release for the next Friday, usually we ship it faster, but since this release contains breaking changes, so I wanted to make sure that everything get documented and reviewed first.

**Reply by ahmad-moussawi** [reply]
I've released it under a beta flag, so you can try it for now, appreciate your feedback.
https://www.nuget.org/packages/SqlKata/3.0.0-beta
